app/templates/components/submission-status.hbs

```diff
@@ -0,0 +1,46 @@
+<style>
+  .fas {
+    display: inline;
+    font-size: 0.8em;
+    opacity: 0.9;
+    width: 15%;
+  }
+  .not-started>.fa-circle {
+    color: #c8ced3;
+  }
+  .Complete>.fa-circle, .Complete .fa-info-circle{
+    color: #4dbd74;
+  }
+  .See.details>.fa-circle {
+    color: #ffc107;
+  }
+  .stalled>.fa-circle {
+    color: #f86c6b;
+  }
+</style>
+
+  <span class="status {{status}} pr-1">
+    {{#if (eq status "Complete")}}
+      <span tooltip-position="left" tooltip="All target repositories have received a copy of the Submission and verified that it is approved for publication">
+        <i class="fas fa-info-circle"></i> {{status}}
+      </span>
+    {{/if}}
+    {{#if (eq status "Manuscript expected")}}
+      <span tooltip-position="left" tooltip="Your funder is aware of this publication and is expecting the deposit of your manuscript.">
+        <i class="fas fa-info-circle"></i> {{status}}
+      </span>
+    {{/if}}
+    {{#if (eq status "Submitted")}}
+      <span tooltip-position="left" tooltip="This submission has either been sent to all target repositories, or is in the queue and will be submitted shortly.">
+        <i class="fas fa-info-circle"></i> {{status}}
+      </span>
+    {{/if}}
+    {{#if (eq status "Stalled")}}
+      <span tooltip-position="left" tooltip="Manual intervention required.">
+        <i class="fas fa-info-circle"></i> {{status}}
+      </span>
+    {{/if}}
```

Comment:
Per the description here, this should be `See details`, there is no stalled status show for a combined status calculations, so this should not be displayed on the Submissions table or at the top of Submission Details page

https://github.com/OA-PASS/pass-ember/issues/568

> * submission.source="pass" && submission.submitted=true && no repoCopy: display submitted
> * submission.source="other" && submission.submitted=false && no repoCopy : display Manuscript expected
> * ALL of repoCopy.copyStatus="complete": display complete
> * Other combination of repositoryCopy.copyStatus: display a link saying "See details" that links to the submission details page. This part relates to #539 which will have a nicer way of displaying all repositoryCopy.copyStatus

Reply:
To clarify - the one at the top should say "see details" (is currently incorrect), the one on the repository should say "stalled" (is currently correct)

![image](https://user-images.githubusercontent.com/4070836/41802608-f72d1534-764f-11e8-95b0-6bcbd4c338fd.png)